xml/System.IdentityModel.Tokens/SecurityTokenElement.xml

```diff
@@ -41,7 +41,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- When this constructor is used, the <xref:System.IdentityModel.Tokens.SecurityTokenElement.GetIdentities%2A> method is not supported by the new instance unless the <xref:System.IdentityModel.Tokens.SecurityTokenElement.​Validate​Token%2A> method is overridden.  
```

Comment:
this must be a weird character thing. they seem identical but when I copied and pasted, the file showed there were differences.